lib/Inst/Inst.cpp

```diff
@@ -850,7 +947,8 @@ Inst *souper::getInstCopy(Inst *I, InstContext &IC,
       }
     }
     if (!Copy) {
-      if (CloneVars && I->Name.find("reserved_") == std::string::npos)
+      if (CloneVars &&
+          I->Name.find("reservedconst_") == std::string::npos)
```

Comment:
Since this magic string is used in multiple places, please factor it out into a constant string and then refer to that. Same for "reservedinst_"